src/pages/PageNotFound/PageNotFound.js

```diff
@@ -0,0 +1,9 @@
+import { pageNotFound } from "../../assets/";
+import "./pagenotFound.css";
+export const PageNotFound = () => {
+  return (
+    <div className="pageNotFound-img">
+      <img src={pageNotFound} alt="" width="760px" />

```

Comment:
can add the alt message here

Reply:
done [2ea6e9e](https://github.com/Manpreet-hub/Tunes/pull/3/commits/2ea6e9e7ef0ffda1c0c297ce3b1d5971502aa1eb)